firehose/firehose.go

```diff
@@ -228,7 +228,7 @@ func (output *OutputPlugin) processRecord(record map[interface{}]interface{}) ([
 	data = append(data, []byte("\n")...)
 
 	if len(data) > maximumRecordSize {
-		return nil, fmt.Errorf("Log record greater than max size allowed by Kinesis")
+		data = data[:maximumRecordSize]
 	}
```

Comment:
@hossain-rayhan @zhonghui12 I think we should should still have some sort of warning in this case, so that the user knows their log was truncated. What do you think?

Reply:
Agree. We can print a warning message here. `logrus.Warnf()`

Reply:
Yes I agree. I will add it